version/news/NEWS-2024.04.2-chocolate-cosmos.md

```diff
@@ -7,6 +7,12 @@
 
 #### Posit Workbench
 
+- Added a Generic Architectures page to the Workbench Admin Guide <i class="bi bi-info-circle-fill" title="Documentation change since last release/patch."></i>
+- Added an AWS Single Server reference architecture to the Workbench Admin Guide <i class="bi bi-info-circle-fill" title="Documentation change since last release/patch."></i>
+- Added an AWS EKS reference architecture to the Workbench Admin Guide <i class="bi bi-info-circle-fill" title="Documentation change since last release/patch."></i>
+- Added an Background Jobs vs Workbench jobs page to the Workbench Admin Guide <i class="bi bi-info-circle-fill" title="Documentation change since last release/patch."></i>
```

Comment:
```suggestion
- Added a Background Jobs vs Workbench jobs page to the Workbench Admin Guide <i class="bi bi-info-circle-fill" title="Documentation change since last release/patch."></i>
```